application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -634,33 +634,45 @@ define('diagram-store', ['jquery', 'xwiki-meta', 'xwiki-utils', 'diagram-utils',
     }
   };
 
+  var getTagName = function(element) {
+    if (!element) { return ''; }
+    return element.tagName.toLowerCase();
+  };
+
+  var isTextElement = function(element) {
+    if (!element) { return false; }
+    return element.nodeType == 3;
```

Comment:
Could also be written as:

    return element && element.nodeType === 3;

In any case, the triple equals operator should be used.